modules/ravestate_nlp/__init__.py

```diff
@@ -101,9 +101,9 @@ def nlp_contains_roboy_signal(ctx):
             return rs.Emit()
         return False
 
-    @rs.state(signal=sig_is_question, read=prop_triples)
+    @rs.state(signal=sig_is_question, read=(prop_triples, prop_tags))
     def nlp_is_question_signal(ctx):
-        if ctx[prop_triples][0].is_question():
+        if ctx[prop_triples][0].is_question() or prop_tags_indicates_question(ctx[prop_tags]):
             return rs.Emit()
```

Comment:
I think we should somehow preserve the information that it might be a yes-no question. My favoured solution would be to add `triple.set_yesno_question()`/`triple.is_yesno_question()` and have `triple.is_question()` also return true if it is a yesno question...

Reply:
that's true - I added this!